spec.bs

```diff
@@ -8,38 +8,40 @@ Repository: dylanvanassche/rdf-ts-snippets-spec
+Abstract: This document specifies how time series data with with only a few changing values
+  can be described in RDF.
 Markup Shorthands: markdown yes, css no
 </pre>
 
-# Introduction # {#intro}
+# Abstract # {#abstract}
 
-Sensor measurements, positioning data, etc. are commonly published as [=Time Series=]
-data with a timestamp and a value. 
-Representing [=Time Series=] data in RDF drastically raises the verbosity,
-which Time Series Snippets try to address by compacting the data points in subsets of a [=Time Series=]
-through [Common Data Types](https://w3id.org/awslabs/neptune/SPARQL-CDTs/spec/latest.html) such as `cdt:List` and `cdt:Map`.
+An RDF Time Series Snippet is a segment of data points from a time series dataset, typically used for analysis or visualization.
+The specification uses [SPARQL Common Data Types](https://awslabs.github.io/SPARQL-CDTs/spec/latest.html) coined by AWS Lab to encode the data points in a values literal.
+The entity can then be used for annotating this snippet, which is a time interval subset of a potentially larger time series.
 
-Note: Time Series Snippets can always be expanded into RDF without the need for Common Data Types.
+# Introduction # {#intro}
 
-The following example shows a set of sensor values with [=Snippet=]s and afterwards expanded to RDF:
+Sensor observations, positioning data, measurements, mobility information... are commonly published as Time Series data with a timestamp and a value. 
+Representing Time Series data in RDF drastically raises the verbosity, as each data point is often given its own identifier, for which a lot of contextual information is repeated on all data points.
+With Time Series Snippets, we allow a data publisher to compact the data points in subsets of a Time Series, called the Snippet, by using [SPARQL Common Data Types](https://awslabs.github.io/SPARQL-CDTs/spec/latest.html) such as `cdt:List` and `cdt:Map`.
+A first example illustrates the features of a `tss:Snippet`:
 
 <div class="example" id="simple_example_snippet">
 ```turtle
-<snippet/2024-01-01>
+<snippet/2026-01-01>
   a tss:Snippet;
   tss:points "[
-    { "time": "2024-01-01T06:00:00Z"^^xsd:dateTime, "value": "5.4"^^xsd:double, "id": "0"^^xsd:integer},
-    { "time": "2024-01-01T06:59:59Z"^^xsd:dateTime, "value": "5.2"^^xsd:double, "id": "1"^^xsd:integer},
-    { "time": "2024-01-01T08:00:00Z"^^xsd:dateTime, "value": "5.2"^^xsd:double, "id": "2"^^xsd:integer},
-    { "time": "2024-01-01T09:00:00Z"^^xsd:dateTime, "value": "6.1"^^xsd:double, "id": "3"^^xsd:integer},
+    { "time": "2026-01-01T06:00:00Z"^^xsd:dateTime, "value": "5.4"^^xsd:double, "id": "0"^^xsd:integer},
+    { "time": "2026-01-01T06:59:59Z"^^xsd:dateTime, "value": "5.2"^^xsd:double, "id": "1"^^xsd:integer},
+    { "time": "2026-01-01T08:00:00Z"^^xsd:dateTime, "value": "5.2"^^xsd:double, "id": "2"^^xsd:integer},
+    { "time": "2026-01-01T09:00:00Z"^^xsd:dateTime, "value": "6.1"^^xsd:double, "id": "3"^^xsd:integer},
```

Comment:
Given that we have `id` now as a term by default according to the spec in this PR, shouldn't the datatype not be something else?

Not sure how we can define 'blank node' as datatype, for URI there's `xsd:anyURI`.